doc/src/advanced_setup.md

```diff
@@ -34,12 +34,13 @@ fn main() {
 
 ### Using the Legacy LALR Parser
 
-By default, LALRPOP uses the
-[lane table](../../lalrpop/src/lr1/lane_table/README.md)
+By default, LALRPOP uses the [lane table][]
 algorithm which is LR(1) but creates much smaller tables. There is no longer
 any clear benefit to using the previous LALR implementation but it is still
 available.
 
+[lane table]: https://github.com/lalrpop/lalrpop/blob/master/lalrpop/src/lr1/lane_table/README.md
```

Comment:
The rendered link is `../../lalrpop/src/lr1/lane_table/README.html` but not `../../lalrpop/src/lr1/lane_table/README.md`.
will `https://github.com/lalrpop/lalrpop/lalrpop/src/lr1/lane_table/README.html` exist?

Reply:
I checked it is correctly rendered as `.md` on deploy build
http://lalrpop.github.io/lalrpop/advanced_setup.html